streaming/src/main/scala/com/stratio/crossdata/streaming/helpers/CrossdataStatusHelper.scala

```diff
@@ -60,61 +77,20 @@ object CrossdataStatusHelper extends SparkLoggerComponent {
                          zookeeperConfiguration: Map[String, String],
                         ephemeralTableName: String): Unit = {
 
-    createEphemeralStatusActor(zookeeperConfiguration, ephemeralTableName)
-
     ephemeralStatusActor.foreach { statusActorRef =>
       statusActorRef ! SetStatus(status)
     }
   }
 
-  //TODO add in the streaming process
-  def checkEphemeralStatus(sparkContext: SparkContext,
-                           streamingContext: StreamingContext,
-                           zookeeperConfiguration: Map[String, String],
-                           ephemeralTableId: String,
-                           ephemeralTableName: String,
-                           stopGracefully: Boolean = StopGracefully): Unit = {
-
-    createEphemeralStatusActor(zookeeperConfiguration, ephemeralTableName)
-
-    ephemeralStatusActor.foreach { statusActorRef =>
-
-      val futureResult = statusActorRef ? GetStatus
-      Await.result(futureResult, timeout.duration) match {
-        case StatusResponse(status) => {
-          if (status == EphemeralExecutionStatus.Stopping) {
-            closeSparkContexts(sparkContext, streamingContext, stopGracefully)
-          }
-        }
-      }
-    }
-  }
 
   def close(): Unit = {
-    for {
-      queryActor <- ephemeralQueryActor
-      statusActor <- ephemeralStatusActor
-    } yield {
-      statusActor ! SetStatus(EphemeralExecutionStatus.Stopped)
-      actorSystem.stop(queryActor)
-      actorSystem.stop(statusActor)
-    }
+
+    ephemeralQueryActor.foreach(_ ! PoisonPill)
+    ephemeralStatusActor.foreach(_ ! PoisonPill)
```

Comment:
do you need set the status Stopped?

statusActor ! SetStatus(EphemeralExecutionStatus.Stopped)